### PR TITLE
DiskRP cli tests 

### PR DESCRIPTION
**Pull request by carissasun**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@carissasun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: test/data/grantAccess.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "access": "Read",
+  "durationInSeconds": 1,
+  "access\"": "Read"
```
Comment (huangpf) [inline test/data/grantAccess.json]: why an additional "access\"" ?

**Comment by azurecla** [comment]
Hi __@carissasun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (caris). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by huangpf** [comment]
@carissasun Can you try to apply for ```cla-not-required``` using the link above?

**Comment by huangpf** [comment]
LGTM. Consider removing the commented test, and turn it into a bug item on the team.

**Comment by balajikris** [comment]
I'm good with this. @carissasun  Let me know if we're ready to merge or if you'd like to respond to @huangpf 's remarks.

**Comment by carissasun** [comment]
I will be addressing these comments and pushing again before merging

File: test/data/grantAccess.json (hunk 1 above)
Comment (carissasun) [inline test/data/grantAccess.json]: mis-typed command, will fix

**Comment by carissasun** [comment]
@balajikris I'm ready to merge, please review

**Comment by balajikris** [comment]
@carissasun -- thanks, looking now..

**Pull request merged**